src/components/GlobalErrorBoundary/index.tsx

```diff
@@ -43,15 +43,16 @@ const LinkContent = styled.div`
   }
 `
 
-export const lastFallbackReloadKey = 'SAFE__lastFallbackReload'
-
 //  When loading app during release, chunk load failure may occur
 export const handleChunkError = (error: Error): boolean => {
+  const MIN_RELOAD_TIME = 10_000
+
   const chunkFailedMessage = /Loading chunk [\d]+ failed/
   const isChunkError = error?.message && chunkFailedMessage.test(error.message)
 
   if (!isChunkError) return false
 
+  const lastFallbackReloadKey = 'SAFE__lastFallbackReload'
```

Comment:
This const should also be on top of the function. I guess it's a matter of taste if we use cap case or not, but it should be consistent.

Reply:
I've adjust this. I like the idea of moving it to the top in uppercase.